HabTrack/app/src/main/java/com/example/habtrack/MainActivity.java

```diff
@@ -29,6 +47,22 @@
 import java.util.HashMap;
 import com.example.habtrack.ui.edithabit.EdithabitFragment;
 
+/**
+ * MainActivty is the starting activity of the HabTrack Application. This class contains onClick
+ * button listening events for the "login" and "signup" button. It also contains method calls
+ * to verify user inputs of email and password if the user attempts to login. And lastly it
+ * contains an instance of the LoginHandler in case of an attempted login with potentially valid
+ * credentials. Depending on the user inputs/ actions a UserProfileActivity or SignUpActivity
+ * activity may be started.
+ *
```

Comment:
'MainActivity' may not be the starting activity, I have made a new PR (#26) which contains `UserLoginStatusActivity` which will be the launch activity. More info you can look into that PR

Reply:
Updated to reflect that MainActivity may not be starting activity.